src/main/resources/assets/sodium/lang/en_us.json

```
@@ -0,0 +1,56 @@
+  "sodium.options.use_compact_vertex_format.name": "Use Compact Vertex Format",
+  "sodium.options.use_compace_vertex_format.tooltip": "If enabled, a more compact vertex format will be used for chunk meshes which limits the precision of vertex attributes. This format will reduce graphics memory usage and bandwidth requirements by around 40%, but could cause z-fighting/flickering texture issues in some edge cases.",
+  "sodium.options.use_fog_occlusion.name": "Use Fog Occlusion",
+  "sodium.options.use_fog_occlusion.tooltip": "If enabled, chunks which are determined to be fully hidden by fog effects will be skipped during rendering. This will generally provide a modest improvement to the number of chunks rendered each frame, especially where fog effects are heavier (i.e. while underwater.)",
+  "sodium.options.use_entity_culling.name": "Use Entity Culling",
+  "sodium.options.use_entity_culling.tooltip": "If enabled, a secondary culling pass will be performed before attempting to render an entity. This additional pass takes into account the current set of visible chunks and removes entities which are not in any visible chunks.",
+  "sodium.options.use_particle_culling.name": "Use Particle Culling",
+  "sodium.options.use_particle_culling.tooltip": "If enabled, only particles which are determined to be visible will be rendered. This can provide a significant improvement to frame rates when many particles are nearby.",
+  "sodium.options.animate_only_visible_textures.name": "Animate Only Visible Textures",
+  "sodium.options.animate_only_visible_textures.tooltip": "If enabled, only animated textures determined to be visible will be updated. This can provide a significant boost to frame rates on some hardware. If you experience issues with some textures not being animated, disable this option.",
+  "sodium.options.use_memory_intrinsics.name": "Use Memory Intrinsics",
+  "sodium.options.use_memory_intrinsics.tooltip": "If enabled, special intrinsics will be used to speed up the copying of client memory in certain vertex-limited scenarios, such as particle and text rendering. This option only exists for debugging purposes and should be left enabled unless you know what you are doing.",
+  "sodium.options.disable_driver_blacklist.name": "Disable Driver Blacklist",
+  "sodium.options.disable_driver_blacklist.tooltip": "If selected, Sodium will ignore the built-in driver blacklist and enable options which are known to be broken with your system configuration. This might cause serious problems and should not be used unless you really do know better. The settings screen must be saved, closed, and re-opened after changing this option in order to reveal previously hidden options.",
+  "sodium.options.performance_impact_string": "%sPerformance Impact: %s",
```

```suggestion
  "sodium.options.performance_impact_string": "Performance Impact: %s",
```
See comment in `SodiumOptionsGUI.java`.